server/src/main/java/org/opensearch/cluster/coordination/PublicationTransportHandler.java

```diff
@@ -377,6 +377,10 @@ private boolean validateRemotePublicationConfiguredOnAllNodes(DiscoveryNodes dis
         return true;
     }
 
+    private void updateLastSeen(final ClusterState clusterState) {
+        lastSeenClusterState.set(clusterState);
+    }
+
```

Comment:
nit: Would prefer inlining it

Reply:
This logic is added in a function so that we can pass this method reference to the `handleApplyCommit` function in Coordinator.